features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.scss

```diff
@@ -0,0 +1,9 @@
+.sub-attribute-row {
```

Comment:
Let's add the license header. Also let's wrap the styles with the class selector of the parent element so that these styles are scoped only to our intended component.

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/8111/commits/d80a15d94de30c9b772046fde6e66df8e10dea37